reference/providers/oracle.md

```diff
@@ -1,8 +1,90 @@
 ---
 layout: single
-title:  "Oracle"
+title:  "Oracle Cloud"
 sidebar:
   nav: reference
 ---
 
 {% include toc %}
+
+For more information about Oracle Cloud or any of the terms used below, please consult
+[Oracle Cloud Infrastructure Documentation](https://docs.cloud.oracle.com/iaas/Content/home.htm).
+
+## Resource mapping
+
+### Account
+The Oracle Cloud provider in Spinnaker maps a Spinnaker [__Account__](/concepts/providers/#accounts)
+to an [Oracle Cloud Infrastructure user]( https://cloud.oracle.com/en_US/tryit) in 
+[Oracle Cloud Infrastructure](https://cloud.oracle.com/) (OCI). 
+Spinnaker authenticates itself with Oracle Cloud using OCI user credentials - 
+see the [setup guide](/setup/providers/oracle). 
+
+### Load Balancer
+A Spinnaker **load balancer** maps to an [OCI Load Balancer](https://docs.cloud.oracle.com/iaas/Content/Balance/Concepts/balanceoverview.htm).
+
+The OCI Load Balancing service offers a load balancer with your choice of a public or private IP
+address, and provisioned bandwidth. The essential components for load balancing include:
+* A load balancer with pre-provisioned bandwidth.
+* A backend set with a health check policy. See [Managing Backend Sets](https://docs.cloud.oracle.com/iaas/Content/Balance/Tasks/managingbackendsets.htm).
+* One or more listeners . See [Managing Load Balancer Listeners](https://docs.cloud.oracle.com/iaas/Content/Balance/Tasks/managinglisteners.htm).
+* Optionally, you can associate your listeners with SSL server certificates to manage how your system handles SSL traffic. See [Managing SSL Certificates](https://docs.cloud.oracle.com/iaas/Content/Balance/Tasks/managingcertificates.htm).
+
+### Server Group
+A Spinnaker **server group** maps to a group of [OCI Virtual Machine Instances](https://docs.cloud.oracle.com/iaas/Content/Compute/Concepts/computeoverview.htm).
+
+### Instance
+A Spinnaker **instance** maps to an [OCI Virtual Machine Instance](https://docs.cloud.oracle.com/iaas/Content/Compute/Concepts/computeoverview.htm).
+
+When you launch an instance, you choose the most appropriate shape which is a template describing 
+the number of CPUs, amount of memory, and other resources allocated to a newly created instance. 
+See [Compute Shapes](https://docs.cloud.oracle.com/iaas/Content/Compute/References/computeshapes.htm) 
+for a list of available VM shapes.
+
+### Firewall
+In OCI, a [security list](https://docs.cloud.oracle.com/iaas/Content/Network/Concepts/securitylists.htm) 
+provides a virtual firewall for an instance, with ingress and egress rules that specify the types 
+of traffic allowed in and out. Each security list is enforced at the instance level.
+
+The Oracle Cloud provider in Spinnaker currently does not support the ability to create firewalls 
```

Comment:
@shihchang , I'm just curious: what happens in the Spinnaker UI if you try to create a firewall?

Reply:
It shows the messag, "We don't currently support the ability to create firewalls for Oracle Cloud from Spinnaker. You can use the Oracle Cloud console to create and manage Security Lists to provide a virtual firewall for an instance."

This is in the following PR
https://github.com/spinnaker/deck/pull/5868/files#diff-f8eb3d3a6ff49ce3d7f0f4084b69c8f2

Reply:
:+1: